config/defaults/config.LINUX_GFORTRAN.mk

```diff
@@ -5,7 +5,7 @@ RM       = /bin/rm -rf
 
 # Fortran compiler and flags
 FF90        = gfortran
-FF90_FLAGS  = -fdefault-real-8 -O2 -fPIC
+FF90_FLAGS  = -fdefault-real-8 -O2 -fPIC -std=f95
```

Comment:
This is likely not within the scope of this PR, but I have read that it is bad practice to use the `-fdefault-real-8` flag. It may not matter for us, but I want to note it.

Reply:
@eirikurj do you know anything about this? Why is it bad practice? We use it pretty much everywhere I think

Reply:
Maybe @bernardopacini can elaborate on this, but I guess this can be considered bad practice by some, since it may not always give the same precision. However, while not perfect, in most of our architectures it should not be an issue and I would argue that this currently safeguards us from potential precision issues. For the current implementation, I would not want to remove it until we have addressed that.

Reply:
I do not have anything formal to refer to about why it is bad practice, I had simply read that in a few places. I agree that for us it is likely far more problematic to take this out than to leave it in.